<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameOptionsDialog</class>
 <widget class="QDialog" name="GameOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="colorsTab">
      <attribute name="title">
       <string>Colors</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>336</width>
            <height>525</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Background color degradation:</string>
           </property>
           <property name="buddy">
            <cstring>backgroundColorDegradationSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="backgroundColorDegradationMultiplierSpinBox">
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="suffix">
            <string>x</string>
           </property>
           <property name="minimum">
            <double>-20.000000000000000</double>
           </property>
           <property name="maximum">
            <double>20.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.500000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="backgroundColorDegradationSpinBox">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="maximum">
            <number>200</number>
           </property>
           <property name="singleStep">
            <number>2</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Grid color degradation:</string>
           </property>
           <property name="buddy">
            <cstring>gridColorDegradationSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="gridColorDegradationMultiplierSpinBox">
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="suffix">
            <string>x</string>
           </property>
           <property name="minimum">
            <double>-20.000000000000000</double>
           </property>
           <property name="maximum">
            <double>20.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.500000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="gridColorDegradationSpinBox">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="maximum">
            <number>200</number>
           </property>
           <property name="singleStep">
            <number>2</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Piece color degradation:</string>
           </property>
           <property name="buddy">
            <cstring>pieceColorDegradationSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="pieceColorDegradationMultiplierSpinBox">
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="suffix">
            <string>x</string>
           </property>
           <property name="minimum">
            <double>-20.000000000000000</double>
           </property>
           <property name="maximum">
            <double>20.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.500000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="pieceColorDegradationSpinBox">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="maximum">
            <number>200</number>
           </property>
           <property name="singleStep">
            <number>2</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Color preset:</string>
           </property>
           <property name="buddy">
            <cstring>colorPresetComboBox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="colorPresetComboBox">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>Custom</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Default</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Black</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Dark</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Clear</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="controlsTab">
      <attribute name="title">
       <string>Controls</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>General</string>
         </property>
         <layout class="QGridLayout" name="gridLayoutGeneral">
          <item row="2" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Pause:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Restart:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Player 2 (left player)</string>
         </property>
         <layout class="QGridLayout" name="gridLayoutPlayer2">
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Move left:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Move right:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Move down faster:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Rotate piece counter-clockwise:</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Rotate piece clockwise:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Drop piece:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Player 1 (right player)</string>
         </property>
         <layout class="QGridLayout" name="gridLayoutPlayer1">
          <item row="0" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Move left:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Move right:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Move down faster:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Rotate piece counter-clockwise:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Rotate piece clockwise:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Drop piece:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="settingsTab">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Key repetition</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0" colspan="3">
           <widget class="QCheckBox" name="movementKeyRepetitionCheckBox">
            <property name="text">
             <string>Activate &amp;movement key repetition</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QCheckBox" name="rotationKeyRepetitionCheckBox">
            <property name="text">
             <string>Activate &amp;rotation key repetition</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Delay (msecs):</string>
            </property>
            <property name="buddy">
             <cstring>delaySlider</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSlider" name="delaySlider">
            <property name="minimum">
             <number>100</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>215</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QSpinBox" name="delaySpinBox">
            <property name="minimumSize">
             <size>
              <width>68</width>
              <height>0</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="minimum">
             <number>100</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>215</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Rate:</string>
            </property>
            <property name="buddy">
             <cstring>rateSlider</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSlider" name="rateSlider">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>50</number>
            </property>
            <property name="value">
             <number>25</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QSpinBox" name="rateSpinBox">
            <property name="minimumSize">
             <size>
              <width>68</width>
              <height>0</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>50</number>
            </property>
            <property name="value">
             <number>25</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Rotation style</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="radioButtonDefalutRotation">
            <property name="text">
             <string>Default</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonClassicalRotation">
            <property name="text">
             <string>Classical</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonClassicalModifiedRotation">
            <property name="text">
             <string>Classical modified</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonOldRotation">
            <property name="text">
             <string>Old</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonDefault">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;Default</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOK">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonApply">
       <property name="text">
        <string>&amp;Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>buttonOK</tabstop>
  <tabstop>buttonCancel</tabstop>
  <tabstop>buttonDefault</tabstop>
  <tabstop>colorPresetComboBox</tabstop>
  <tabstop>rotationKeyRepetitionCheckBox</tabstop>
  <tabstop>delaySlider</tabstop>
  <tabstop>delaySpinBox</tabstop>
  <tabstop>rateSlider</tabstop>
  <tabstop>rateSpinBox</tabstop>
  <tabstop>radioButtonDefalutRotation</tabstop>
  <tabstop>radioButtonClassicalRotation</tabstop>
  <tabstop>radioButtonClassicalModifiedRotation</tabstop>
  <tabstop>radioButtonOldRotation</tabstop>
  <tabstop>movementKeyRepetitionCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
